### DISCUSSION TITLE
Superset Tutorials on YouTube

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#13256] [General]
Hello community!

If you have not checked out our product tutorial channel(maintained by Preset) on YouTube, please do! :) 
[User's Corner - Short demos and videos to improve your Superset skills](https://www.youtube.com/channel/UCzg8opP7sG8n0Mi0e8yeqAg/featured)

We have over 60 videos uploaded in this channel, covering topics like starter guide, new features, data viz, and frequently asked workarounds. 

As we are continually adding more tutorials, we want YOU to let us know what is missing or what you are most interested in learning. Please reply to the thread for any tutorial requests! Thank you for choosing Superset as your BI tool! 😄

**Comment by junlincc** [comment]
e.g. 
please do a tutorial of the new time-picker in Explore or dashboard cascading filter  or anything

**Reply by eugeniamz** [reply]
[done!](https://www.youtube.com/watch?v=Nnr9AF7aPVA&feature=youtu.be)

**Comment by kamalkeshavani-aiinside** [comment]
I am not sure if it is possible now using CustomSsoSecurityManager, but if possible then would love to see 'How to allow multiple login methods in Superset'.
For example, allow login via Gmail and AUTH_DB together.

**Reply by eugeniamz** [reply]
@srinify & @garden-of-delete maybe you can do it? I just know everything about using Superset no so much about configuration.

**Comment by kamalkeshavani-aiinside** [comment]
Please do a tutorial on Superset Metadata database migration from sqlite to mysql/postgres(copying old data).

**Reply by eugeniamz** [reply]
@srinify & @garden-of-delete maybe you can do it? I just know everything about using Superset no so much about configuration.

**Comment by garden-of-delete** [comment]
Bumping for visibility

**Comment by raman0006** [comment]
Hi,
Any reference that will help me out.
I need to convert the development superset server into a production superset server. I went through all the resources But I couldn't find up to mark.
Any idea how to config the standalone ubuntu server into the superset production.

**Reply by garden-of-delete** [reply]
Hi @raman0006 . I'm not sure how well all the steps there are documented. Might be a good topic to expand on in a blog or the official documentation. I will look into it for you.

**Comment by kamalkeshavani-aiinside** [comment]
@eugeniamz Please do a tutorial on how to use ENABLE_ACCESS_REQUEST feature.
